src/relayclient/GSNConfigurator.ts

```diff
@@ -79,7 +79,7 @@ export function getDependencies (config: GSNConfig, provider?: HttpProvider, ove
   let accountManager = overrideDependencies?.accountManager
   if (accountManager == null) {
     if (provider != null) {
-      accountManager = new AccountManager(provider, defaultEnvironment.chainId, config)
+      accountManager = new AccountManager(provider, config.chainId ?? defaultEnvironment.chainId, config)
```

Comment:
ChainID is not optional in GSNConfig. It will never be null, and will never reach the `defaultEnvironment.chainId`.

Reply:
Except for all the times that it was null and reached it when I integrated with Metacoin.